Directory.Build.props

```diff
@@ -0,0 +1,11 @@
+<Project>
+    <PropertyGroup>
+        <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
+    </PropertyGroup>
+
+    <PropertyGroup>
+        <Avalonia>0.10.7</Avalonia>

```

Comment:
I think this should be `0.10.*`

Reply:
```
error NU1011: Centrally defined floating package versions are not allowed
```

After fighting with `ManagePackageVersionsCentrally` for a few hours now, I think this is the last straw 😅 

I'll try not to get enticed by preview features in the future. I'll rework this PR to focus on improved ScottPlot version bumping and not worry about centralizing versions of dependencies.

I bet we can achieve the `*` behavior with a shared project variable...